shared/constants/search.js

```diff
@@ -7,10 +7,10 @@ export type SearchPlatforms = 'Keybase' | 'Twitter' | 'Github' | 'Reddit' | 'Coi
 
 export type ExtraInfo = {
   service: 'external',
-  icon: IconType,
+  icon: ?IconType,
   serviceUsername: string, // i.e. with twitter it would be malgorithms
-  serviceAvatar: ?string, // i.e. with twitter it would be their twitter avatar url
-  fullNameOnService: ?string // Say with twitter we know malgorithms is "Chris Coyne"
+  serviceAvatar: string, // i.e. with twitter it would be their twitter avatar url
+  fullNameOnService: string // Say with twitter we know malgorithms is "Chris Coyne"
```

Comment:
I think we should add another variant that will be used when we only show the icon, username, and full name. We could call it `externalHint`

This variant would change to only contain avatar, username, and fullname.